src/herder/HerderImpl.cpp

```diff
@@ -148,19 +148,37 @@ void
 HerderImpl::valueExternalized(uint64 slotIndex, StellarValue const& value)
 {
     const int DUMP_SCP_TIMEOUT_SECONDS = 20;
+    bool newSlot = slotIndex == getCurrentLedgerSeq();
```

Comment:
why this is not
* the same value than what is being computed in SCPDriver?
* a parameter (named "isLatestSlot"?)

Reply:
SCPDriver always updates tracking value for new slots. From what I see, HerderImpl::valueExternalized is only called when Herder is back in sync (and this is enforced by trackingSCP in trackingHeartbeat). If you think it is better to just rely on a parameter, it can be changed (isLatestSlot parameter was my initial implementation anyway; but I found it easier to reason about tracking values when in sync or out of sync, so I'd like to understand why using a parameter would be better here)

Reply:
Reason I thought it was a parameter was that `HerderSCPDriver::valueExternalized` seems to be the one defining the criteria for considering processing that slot as "latest" or not.

I guess we have a hidden ordering dependency already so we can leave it like that.